tests/Aspire.Components.Common.Tests/ConformanceTests.cs

```diff
@@ -98,6 +98,7 @@ public void OptionsTypeIsSealed()
     [InlineData(false)]
     public void HealthChecksRegistersHealthCheckService(bool enabled)
     {
+        SkipIfRequiredServerConnectionCanNotBeEstablished();
```

Comment:
Assuming these were all already skipped for a different reason?

Reply:
Oh no, I understand now, these were not run because they didn't have the `RunOnHelix` property.

Reply:
IIUC, these tests were passing on CI because they don't actually use the service. But in cases where we use a testcontainer, we get the ConnectionString from that:
https://github.com/radical/aspire/blob/3468c8513a7a9a979e322595d1da8b482482041e/tests/Aspire.StackExchange.Redis.Tests/ConformanceTests.cs#L66-L74

```csharp
    protected override void PopulateConfiguration(ConfigurationManager configuration, string? key = null)
    {
        string connectionString = RequiresDockerTheoryAttribute.IsSupported
                                    ? _containerFixture.GetConnectionString()
                                    : "localhost";
        configuration.AddInMemoryCollection([
            new KeyValuePair<string, string?>(ConformanceTests.CreateConfigKey("Aspire:StackExchange:Redis", key, "ConnectionString"), connectionString)
        ]);
```

Digging into this, the above is what redis uses so it would be fine without the `SkipIf*`. But others like `npgsql` don't, so they break. But I can have them default to `localhost` on windows too.

I'll remove these skips.

Reply:
fixed.